content/14.biological-interpretation.md

```diff
@@ -1,8 +1,132 @@
+The MCL (Markov CLustering) algorithm is suitable for protein networks in most situations . 
+On the other hand, the Molecular COmplex DEtection (MCODE) algorithm helps detect very densely connected nodes, thus unveiling protein complexes [@PMID:12525261]. 
+In this regard, network clustering is useful for tentatively assigning the function of an uncharacterized protein. 
+If the protein appears in a cluster, its function should be closely related to the cluster members, a principle known as "guilty by association." [@PMID:26215734]
+
+### Network visualization
+
+A critical step in network analysis is to display the data in a structured and uncluttered graph. 
+Networks can rapidly become a hairball unamenable to interpretation. 
+Software platforms like Cytoscape can be used to visualize networks orderly by applying layout algorithms and format styles [@PMID:14597658]. 
+Since many of these platforms are open source, community-designed plugins enhance their capabilities. 
+In Cytoscape, the stringApp adds a search bar to query the STRING database with accession numbers or protein names [@PMID:36512705]. 
+The network is directly retrieved into Cytoscape, where its built-in network analyzer can be used to calculate centralities. 
+Moreover, user-defined information, like fold-change values, can be integrated and mapped into the network.
+
+### Functional enrichment analysis
+
+Gene and gene products have shared attributes regarding their cellular localization, molecular function, or biological process in which they take part, among others.
+For example, citrate synthase and malate dehydrogenase are _mitochondrial_ enzymes that show _RNA binding_ activity and participate in the _Krebs cycle_. 
+The terms in italics represent a fraction of the body of knowledge about these proteins. 
+Similar information can be compiled and assigned to every gene or gene product, forming its gene ontology (GO): a set of concepts that describe the entity. 
+Functional enrichment analysis aims to reveal if genes or gene products grouped under a GO term (a.k.a., a "gene set") are over- or under-represented in a more extensive set. 
+For example, proteomics analysis of human cardiac 3D microtissue exposed to anthracyclines (drugs used in cancer chemotherapy) unearthed several proteins with altered levels [@PMID:33060801]. 
+Many of these were specifically grouped under GO terms related to mitochondrial dysfunction, indicating the detrimental effects of these drugs on the organelle. 
+GO terms [@PMID:10802651] or descriptors from other annotation libraries (like KEGG [@PMID:10592173] or REACTOME [@PMID:34788843]) can be retrieved from STRING when constructing a network or from other freely available compendiums. 
+Then, it is a matter of conducting a functional enrichment test to analyze which terms are over- or under-represented.
+
+#### Over-representation analysis
```

Comment:
this section seems redundant with the previous section - can you combine the two?

Reply:
I kept the three sub-sections as this better reflects the three most widely used types of functional enrichment methods (ORA, FCS, and PT). Instead, I tried to reduce redundancy by rewriting part of the introductory paragraph.